package.json

```diff
@@ -7,38 +7,21 @@
     "node": ">=12.1.0"
   },
   "scripts": {
-    "build-iframe": "node --max_old_space_size=5120 ./node_modules/@angular/cli/bin/ng build --project=ng-zorro-antd-iframe --prod --base-href ./",
-    "build-site": "node --max_old_space_size=5120 ./node_modules/@angular/cli/bin/ng build --prod",
-    "build-site-server": "node --max_old_space_size=5120 ./node_modules/@angular/cli/bin/ng run ng-zorro-antd-doc:server:production",
-    "build:client-and-server-bundles": "npm run build-site && npm run build-site-server",
-    "build:docs": "npm run build:client-and-server-bundles && npm run build:prerender && npm run build-iframe && npm run helper",
-    "build:prerender": "npm run compile:prerender && npm run generate:prerender",
-    "compile:prerender": "tsc -p scripts/prerender/tsconfig.json",
-    "generate:prerender": "cd dist && node prerender && node sitemap",
+    "start": "gulp start:dev",
+    "test": "ng test --watch=false --code-coverage",
```

Comment:
Add a `"test:watch": "ng test --watch=true"` here for convenience?

Reply:
`npm test:watch [button]`